modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -47,7 +47,17 @@ When this alert presents, if the currently set language is one of the xref:bundl
 
 The {productname} 6.7.0 release includes an accompanying release of the **Advanced Code** premium plugin.
 
-**Advanced Code** 3.3.1 includes the following bug fix:
+**Advanced Code** 3.3.1 includes the following improvement and bugfix:
+
+==== Going back from a view to the editor in mobile caused an error.
+// TINY-10003
+In previous versions of {productname}, the toolbar implemented for mobile lacked the essential refresh API.
+
+Consequently, when switching from the **Advanced Code** view back to the editor, {productname} triggered an unintended refresh operation, leading to a TypeError to display within the console.
+
+**Advanced Code** 3.3.1 addresses this issue, by integrating the refresh API into the mobile toolbar.
```

Comment:
```suggestion
**Advanced Code** 3.3.1 addresses this issue, by integrating an empty refresh API into the mobile toolbar.
```

I would change in this way because at the moment that function does nothing as you can see [here](https://github.com/tinymce/tinymce/pull/9007/files#diff-b0cee87bd6b24cdea12593e1658b0972ed489f364882e0a877f3d85b61bed5a7R46)